### ISSUE TITLE
Query and Mutation imports are missing

### ISSUE THREAD

**Issue by DimaSol** [graphql-java-generator/graphql-maven-plugin-project#132]
When generating client code, Query and Mutation imports are missing from QueryExecutor and MutationExecutor class.
This is a degradation in 1.18.4 version.

**Comment by etienne-sf** [comment]
For instance, every thing was fine with version 1.18.5, and this plugin parameters :


```xml
			<plugin>
				<groupId>com.graphql-java-generator</groupId>
				<artifactId>graphql-maven-plugin</artifactId>
				<version>${graphql-maven-plugin.version}</version>
				<executions>
					<execution>
						<goals>
							<goal>generateClientCode</goal>
						</goals>
					</execution>
				</executions>
				<configuration>
					<!-- Let's configure the GraphQL Gradle Plugin: -->
					<!-- All available parameters are described here: -->
					<!-- https://graphql-maven-plugin-project.graphql-java-generator.com/graphql-maven-plugin/generateClientCode-mojo.html -->
					<packageName>org.forum.client</packageName>
					<customScalars>
						<customScalar>
							<graphQLTypeName>Date</graphQLTypeName>
							<javaType>java.util.Date</javaType>
							<graphQLScalarTypeStaticField>com.graphql_java_generator.customscalars.GraphQLScalarTypeDate.Date</graphQLScalarTypeStaticField>
						</customScalar>
					</customScalars>
					<!-- The parameters below change the 1.x default behavior. They are set to respect the behavior of the future 2.x versions -->
					<copyRuntimeSources>false</copyRuntimeSources>
					<separateUtilityClasses>true</separateUtilityClasses>
					<skipGenerationIfSchemaHasNotChanged>true</skipGenerationIfSchemaHasNotChanged>
				</configuration>
			</plugin>
```

**Comment by markxj** [comment]
I also just noticed this when I wanted to upgrade to 1.18.5- I believe it only occurs when separateUtilityClasses is set to true

**Comment by etienne-sf** [comment]
But what is the issue there ?

The main change in the 1.18.5 is precisely to remove these imports, and have all POJO be fully qualified when used. 
This prevents some clashes with Java names.

**Comment by markxj** [comment]
When separateUtilityClasses is set to true, then the QueryExecutor and MutationExecutor are created in the util sub package. So the use of classSimpleName at lines 215, 252, 295, 318 and 357 in client_query_mutation_executor.vm give an unknown symbol

**Comment by etienne-sf** [comment]
Yes. 

Because this use of an import would generate a clash name when a type from the GraphQL schema has the same name as one of the class imported for the 'regular' code of the Executor. 

To cover these cases, the only way was to remove the import. 
And use full classified names instead. 


I ask you again: what is your plug-in configuration ?

As you didn't provide it, I must guess it, in order to help. 
I guess that you're using a specific template for the Executor, right ?
If yes, you'll have to report these changes from the standard template into your custom one.

**Comment by etienne-sf** [comment]
Thank you for this precision. 
This is the missing part in the original post. 



The issue is the combination of generateDeprecatedRequestResponse=false and separateUtilityClasses=true. 

There was no such tests, in all the tested combinations!
:( 

I'll correct that soon


In the meantime, a workaround is to set generateDeprecatedRequestResponse to true.

**Comment by etienne-sf** [comment]
Thé correction for this issue is available in the 1.18.6 release
